src/pybind/mgr/dashboard/frontend/src/app/core/auth/user-form/user-form.component.ts

```diff
@@ -128,10 +129,11 @@ export class UserFormComponent extends CdForm implements OnInit {
     const observables = [this.roleService.list(), this.settingsService.getStandardSettings()];
     observableForkJoin(observables).subscribe(
       (result: [UserFormRoleModel[], CdPwdExpirationSettings]) => {
-        this.allRoles = _.map(result[0], (role) => {
-          role.enabled = true;
-          return role;
-        });
+        // this.allRoles = _.map(result[0], (role) => {
+        //   role.enabled = true;
+        //   return role;
+        // });
```

Comment:
why comment ?

Reply:
commented code is remove in updated commit